### DISCUSSION TITLE
Obsidian and github synchronization!

### DISCUSSION THREAD

**Discussion by spandankeshav** [BryanJenksCommunity/FAQ#170] [Q&A] [answered]
Hello, I have been following Bryans obsidian and git instructions from YouTube and managed to install and also access from my first machine. However, if I want to access my vault from other machine, do I need to follow exactly same procedure as for the 1st machine? Thanks 👍

**Comment by tallguyjenks** [comment, marked as answer]
if you're not using the obsidian sync service they you would first need to clone the repository to the new machine, then yes, setup the github syncing workflow so that it will sync changes from the new machine as well as the old machine to the centralized location of github
